nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Rb post auth

Fixed bug that allowed non admin users to see unapproved or unpublished posts that weren't their own

Testing:
- Login as a non admin user
- go to "/post/{id}" for several posts
- If the post isn't the users AND it is either unapproved or unpublished, there should be a message saying "This isn't the post you're looking for"
- Otherwise, it should direct you to the post details
- Check to make sure admins can see all posts
- Check to see if GET requests made on Postman return Unauthorized for the above reasons
